### ISSUE TITLE
Online softmax error when using RTX 5070 Ti and Fish OpenAudio S1-mini

### ISSUE THREAD

**Issue by ZobaJakColbert** [zeropointnine/tts-audiobook-tool#4]
Hi, I'm using RTX 5070 Ti and Fish OpenAudio S1-mini. Also using this command to install torch

**CUDA 12.8**
```
pip install torch==2.7.1 torchvision==0.22.1 torchaudio==2.7.1 --index-url https://download.pytorch.org/whl/cu128
```
but when trying to generate audio this error is happening

```
Initializing whisper model (large-v3, cuda, float16)...


[1/8] Generating audio for text segment 1:
Introduction

  0%|                                                                                                                                                          | 0/7600 [00:00<?, ?it/s]D:\Python\Audiobook from eBook Creator 2025\tts-audiobook-tool\venv-fish\Lib\site-packages\torch\_inductor\lowering.py:7007: UserWarning:
Online softmax is disabled on the fly since Inductor decides to
split the reduction. Cut an issue to PyTorch if this is an
important use case and you want to speed it up with online
softmax.

  warnings.warn(
```

**Comment by zeropointnine** [comment]
Hi, that warning from the model can be ignored. 

On the very first inference, Fish goes through a compile step, which takes about one minute. I will should add a one-time alert about this behavior.

Did you wait long enough before giving up on it?

5070Ti with the torch install command you pasted ought to be good.

**Comment by ZobaJakColbert** [comment]
Yeah, I've got this. It was a bug on Windows with pytorch dll.

Finally I'm running fish with 
```
pip install torch==2.8 torchvision torchaudio==2.8 --index-url https://download.pytorch.org/whl/cu129
```

and also fixed torch_python.dll with [Strox0](https://github.com/Strox0) comment
https://github.com/pytorch/pytorch/issues/162430#issuecomment-3289054096
```
For those who want to fix the issue by patching the image file here are the steps to do so. It is quite simple.

Download a hex editor like HxD
Find 'torch_python.dll', located in '...\Lib\site-packages\torch\lib'
Open a command prompt and type in 'tasklist /m torch_python.dll', if any processes come up they are currently using the DLL and they need to be shutdown.
Create a copy of the DLL just in case anything goes wrong.
Open the DLL with your hex editor
Search for the string 'KiiiiisOl'
You need to replace the last character 'l' byte '6C' with 'K' byte '4B'
Save the DLL.
Done. Next time you launch torch it will load the modified DLL and the issue above mentioned will be solved.
```
